api/overall_status_test.go

```diff
@@ -86,24 +86,27 @@ func TestOverallStatus_TaskStatusToOverall(t *testing.T) {
 		expected string
 	}{
 		{
-			"healthy: all healthy",
```

Comment:
These comments aren't accurate anymore. I'll likely remove this test all together so just doing the renaming for now